common/screenshotter.go

```diff
@@ -276,13 +287,24 @@ func (s *screenshotter) screenshotElement(h *ElementHandle, opts *ElementHandleS
 	}
 
 	documentRect := bbox
-	rt := h.execCtx.vu.Runtime()
 	scrollOffset := h.Evaluate(`() => { return {x: window.scrollX, y: window.scrollY};}`)
-	switch s := scrollOffset.(type) {
-	case goja.Value:
-		documentRect.X += s.ToObject(rt).Get("x").ToFloat()
-		documentRect.Y += s.ToObject(rt).Get("y").ToFloat()
+
+	so, ok := scrollOffset.(map[string]interface{})
+	if !ok {
+		return nil, fmt.Errorf("converting scroll offset %q: %w", scrollOffset, err)
+	}
+
+	x, ok := so["x"].(float64)
+	if !ok {
+		return nil, fmt.Errorf("converting scroll offset x %q: %w", so["x"], err)
+	}
+	documentRect.X += x
+
+	y, ok := so["y"].(float64)
+	if !ok {
+		return nil, fmt.Errorf("converting scroll offset y %q: %w", so["y"], err)
 	}
+	documentRect.Y += y
```

Comment:
Same. We can simply use the `convert` helper here.

Reply:
Resolved in 5e38ec56b04eabc4a02f05d8230d784946520ede